model/src/test/kotlin/licenses/LicenseViewTest.kt

```diff
@@ -55,23 +51,12 @@ class LicenseViewTest : WordSpec() {
 
     private fun containLicensesWithSources(
         vararg licenses: Pair<String, LicenseSource>
-    ): Matcher<List<Pair<SpdxExpression, LicenseSource>>?> =
-        neverNullMatcher { value ->
-            val expectedLicenses = licenses.map { it.first.toSpdx() }.toSet()
-            val actualLicenses = value.map { it.first }.toSet()
-
-            if (expectedLicenses == actualLicenses) {
-                containAll(licenses.map { Pair(it.first.toSpdx(), it.second) }).test(value)
-            } else {
-                MatcherResult(
-                    false,
-                    {
-                        "List should contain exactly licenses ${expectedLicenses.print().value}, but has " +
-                                actualLicenses.print().value
-                    },
-                    { "List should not contain exactly licenses ${expectedLicenses.print().value}" }
-                )
-            }
+    ): Matcher<List<Pair<SpdxSingleLicenseExpression, LicenseSource>>?> =
+        transformingCollectionMatcher(
+            expected = licenses.toList(),
+            matcher = ::containAll
+        ) { list ->
+            list.map { (expression, source) -> expression.toString() to source }
```

Comment:
I'm a bit confused where the `toSpdx()` conversion from before did go... and why we now have `toString()` instead. Could you please elaborate and / or explain in the commit message?

Reply:
I have added an explanation to the commit message.